core/Files.h

```diff
@@ -104,6 +110,8 @@ class File final {
 
     Flags flags;
 
+    NameRef package = core::NameRef::noName();
```

Comment:
This is what's causing the bump in size for `File`.

Reply:
Do you think it's worth leaving a comment here like "this incurs a memory overhead even if the `--stripe-packages` flag is not passed at the command line. In the future we may want to consider ways to pay this cost only if that flag is passed at the command line."

Reply:
I can add a note to that effect 👍